internal/backend/akmodules/os/os.go

```diff
@@ -0,0 +1,98 @@
+package os
+
+import (
+	"context"
+	"errors"
+	"os/exec"
+	"runtime"
+	"time"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/fixtures"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkexecutor"
+	"go.autokitteh.dev/autokitteh/sdk/sdkmodule"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+var ExecutorID = sdktypes.NewExecutorID(fixtures.NewBuiltinIntegrationID("os"))
+
+func New() sdkexecutor.Executor {
+	return fixtures.NewBuiltinExecutor(
+		ExecutorID,
+		sdkmodule.ExportFunction("command", command, sdkmodule.WithArgs("cmd", "*args")),
+		sdkmodule.ExportFunction("shell", shell, sdkmodule.WithArgs("cmd", "sh?")),
+	)
+}
+
+func execute(ctx context.Context, name string, args ...string) (sdktypes.Value, error) {
```

Comment:
This is pretty much a duplication of the code in `integrations`, any way to combine them?

Reply:
mistake.